src/Libraries/Samples/SampleIntegration/TracedHog.cs

```diff
@@ -69,7 +71,7 @@ public HogID(SerializationInfo info, StreamingContext context)
 
 
 [DSNodeServices.RegisterForTrace]
-public class TracedHog
+public class TracedHog : IDisposable
 {
```

Comment:
The document is written to be very incremental in adding features one at a time to the class. So I think we might consider a 'TracedLifeCycleManagedHog' for adding dispose services.

What do you think?